src/common/file_windows.c

```diff
@@ -211,3 +211,14 @@ util_file_device_dax_alignment(const char *path)
 
 	return 0;
 }
+
+/*
+ * util_get_dev_id -- returns dev_id from file stat
+ */
+os_dev_t
```

Comment:
I don't this will EVER be relevant on windows. Deep flush will probably just amount to msync (or some ioctl).
All the more reason for a proper abstraction layer.